test/performance/odp_packet_gen.c

```diff
@@ -2290,7 +2290,7 @@ int main(int argc, char **argv)
 
 	if (global->test_options.direct_rx)
 		drain_direct_input(global);
-	else
+	else if (global->test_options.num_rx)
 		drain_scheduler(global);
```

Comment:
A minor thing, but I'd move this check inside `drain_scheduler()`, like it's already done in `drain_direct_input()`, so this if else stays simpler.

Reply:
Ack